tests/test_serialize.py

```diff
@@ -81,7 +92,11 @@ def test_wrong_device_name():
         (qml.Hadamard(0), NamedObsC128),
         (qml.Hermitian(np.eye(2), wires=0), HermitianObsC128),
         (
-            qml.PauliZ(0) @ qml.Hadamard(1) @ (0.1 * (qml.PauliZ(2) + qml.PauliX(3))),
+            (
+                qml.PauliZ(0) @ qml.Hadamard(1) @ (0.1 * (qml.PauliZ(2) + qml.PauliX(3)))
```

Comment:
Why the different observable?

Reply:
`lightning.tensor` does not support creating a tensor product obs from Hamiltonain obs.